src/states/elder/mod.rs

```diff
@@ -1304,7 +1323,7 @@ impl Elder {
         let src = Authority::Section(vote.dst_name);
         let dst = vote.old_client_auth;
         let content = MessageContent::RelocateResponse {
-            target_interval: target_interval,
+            target_interval: (target_interval.0, target_interval.1),
```

Comment:
Why not change the type of this field to `XorTargetInterval` too?

Reply:
It was slowly getting everywhere, I did not think it was warranted at this point to update the message and all its handlers.
If you think it is, I can add an extra commit?